WordPress/src/main/java/org/wordpress/android/ui/notifications/services/NotificationsPendingDraftsService.java

```diff
@@ -0,0 +1,229 @@
+package org.wordpress.android.ui.notifications.services;
+
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.Context;
+import android.content.Intent;
+import android.os.IBinder;
+import android.support.v4.app.NotificationCompat;
+
+import org.wordpress.android.R;
+import org.wordpress.android.WordPress;
+import org.wordpress.android.models.Post;
+import org.wordpress.android.push.NativeNotificationsUtils;
+import org.wordpress.android.push.NotificationsProcessingService;
+import org.wordpress.android.ui.main.WPMainActivity;
+import org.wordpress.android.util.AppLog;
+
+import java.util.ArrayList;
+
+import static org.wordpress.android.push.GCMMessageService.GENERIC_LOCAL_NOTIFICATION_ID;
+
+
+public class NotificationsPendingDraftsService extends Service {
+
+    private boolean running = false;
+    public static final int PENDING_DRAFTS_NOTIFICATION_ID = GENERIC_LOCAL_NOTIFICATION_ID + 1;
+    public static final String POST_ID_EXTRA = "postId";
+    public static final String IS_PAGE_EXTRA = "isPage";
+    private static final long MINIMUM_ELAPSED_TIME_BEFORE_REPEATING_NOTIFICATION = 24 * 60 * 60 * 1000; //a full 24 hours day
+    private static final long MAX_DAYS_TO_COUNT_DAYS = 30; //30 days
```

Comment:
What do you think on renaming this constant to something like `MAX_DAYS_TO_SHOW_DAYS_IN_MESSAGE` ?
I know it's not that better, but at least there is the purpose of the constant in its name.